app/src/gecko/java/org/mozilla/focus/web/GeckoViewPrompt.java

```diff
@@ -285,7 +304,8 @@ private void addChoiceItems(final int type, final ArrayAdapter<Choice> list,
         }
     }
 
-    public void promptForChoice(final GeckoSession session, final String title,
+    @Override
+    public void onChoicePrompt(final GeckoSession session, final String title,
```

Comment:
\[PMD\] INFO: The method 'onChoicePrompt' has a Modified Cyclomatic Complexity of 33\.